README.rst

```diff
@@ -18,7 +18,7 @@ pyca/cryptography
 
 ``cryptography`` is a package which provides cryptographic recipes and
 primitives to Python developers.  Our goal is for it to be your "cryptographic
-standard library". It supports Python 2.7, Python 3.6+, and PyPy 5.4+.
+standard library". It supports Python 3.6+ and PyPy3 5.4+.
```

Comment:
This is untrue. We only support PyPy3 with 3.6+ compat at this point (or at least we'll break PyPy3 <= that version when we add type annotations). This is probably 7.x?

Reply:
Looks like 7.2

Reply:
* PyPy v7.0.0 was the first to support 3.6 as an alpha
https://morepypy.blogspot.com/2019/02/pypy-v700-triple-release-of-27-35-and.html

* PyPy v7.1 was the first to support 3.6 as a beta
https://morepypy.blogspot.com/2019/03/pypy-v71-released-now-uses-utf-8.html

* And PyPy v7.2 was the first with full 3.6 support
https://morepypy.blogspot.com/2019/10/pypy-v72-released.html